osu.Game/Overlays/Profile/ProfileHeader.cs

```diff
@@ -328,7 +310,8 @@ private void loadUser()
                 Depth = float.MaxValue,
             }, coverContainer.Add);
 
-            if (user.IsSupporter) supporterTag.Show();
+            if (user.IsSupporter)
+                SupporterTag.Show();

```

Comment:
does this not need a hide?

Reply:
The ProfileHeader is currently recreated with each user that is to be shown if I read this correctly.

Reply:
I see.. kinda weird it has a user property in the first place then but let's go with it